.github/steps/1-create-a-workflow.md

```diff
@@ -20,24 +12,30 @@ _Welcome to "Hello GitHub Actions"! :wave:_
 - To read more about workflows, jobs, and events, see "[Understanding GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions)".
 - If you want to learn more about the `pull_request` event before using it, see "[pull_request](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request)".
 
-To get you started, we used actions to go ahead and made a branch and pull request for you.
+To get you started, we ran an Actions workflow in your new repository that, among other things, created a branch for you to work in, called `welcome-workflow`.
 
 ### :keyboard: Activity: Create a workflow file
 
 1. Open a new browser tab, and navigate to this same repository. Then, work on the steps in your second tab while you read the instructions in this tab.
-1. Create a pull request to view all the changes you'll make throughout this course. Click the **Pull Requests** tab, click **New pull request**, set `base: main` and `compare:welcome-workflow`, click **Create pull request**.
+1. Create a pull request. This will contain all of the changes you'll make throughout this part of the course.
+
+   Click the **Pull Requests** tab, click **New pull request**, set `base: main` and `compare:welcome-workflow`, click **Create pull request**, then click **Create pull request** again.
+
 1. Navigate to the **Code** tab.
 1. From the **main** branch dropdown, click on the **welcome-workflow** branch.
 1. Navigate to the `.github/workflows/` folder, then select **Add file** and click on **Create new file**.
-1. In the **Name your file...** field, enter `welcome.yml`.
+1. In the **Name your file** field, enter `welcome.yml`.
 1. Add the following content to the `welcome.yml` file:
-   ```yaml
+
+   ```yaml copy
    name: Post welcome comment
    on:
      pull_request:
        types: [opened]
    permissions:
      pull-requests: write
    ```
-1. To commit your changes, click **Commit new file**.
-1. Wait about 20 seconds for actions to run, then refresh this page (the one you're following instructions from) and an action will automatically close this step and open the next one.
+
+1. To commit your changes, click **Commit changes**.
+1. Type a commit message, select **Commit directly to the welcome-workflow branch** and click **Commit changes**.
+1. Wait about 20 seconds, then refresh this page (the one you're following instructions from). A separate Actions workflow in the repository (not the workflow you created) will run and will automatically replace this content with instructions for the next step.
```

Comment:
I'm wondering if we should mention that this is the `README` file?

Reply:
[]()